mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -200,6 +218,168 @@ func resourceDataprocCluster() *schema.Resource {
 							Description:  `The Cloud Storage temp bucket used to store ephemeral cluster and jobs data, such as Spark and MapReduce history files. Note: If you don't explicitly specify a temp_bucket then GCP will auto create / assign one for you.`,
 						},
 
+<% if version == "ga" -%>
+						"gke_cluster_config": {
```

Comment:
This seems to combine VirtualClusterConfig, GKEClusterConfig, GKENodePoolConfig and potentially more (it's hard to follow) from the API. We aim to be as in line as possible to the API schema and deviating from that could lead to challenges updating the resource in the future as schema changes are breaking changes. What is the reasoning here?

Reply:
makes sense! I wanted to simplify the structure and make it easier for users to use. But I get your point. I will change this according to the API